packages/contracts/contracts/discovery/Controller.sol

```diff
@@ -129,10 +132,57 @@ contract Controller is IController, AccessControl {
 
     /// @inheritdoc IController
     function getBatchForProject(address _project)
-        external
+        public
         view
         returns (address)
     {
         return projectsToBatches[_project];
     }
+
+    function vote(
+        address _project,
+        uint256 _peoplesPoolAmount,
+        uint256 _stakersPoolAmount
+    ) external {
+        // TODO require KYC
+
+        Batch batch = getBatchForProject(_project);
+
+        IBatch(batch).vote(_project, _peoplesPoolAmount, _stakersPoolAmount);
```

Comment:
The batch's vote does not take amounts anymore. The controller is fully in charge of this.